recipe/meta.yaml

```diff
@@ -38,13 +38,13 @@ requirements:
     - setuptools
     - mysql-devel {{ version }}
     - libprotobuf
-    - protobuf >=3.0.0
+    - protobuf >=3.11.0,<=3.20.1
     - zlib
 
   run:
     - python
-    - protobuf >=3.0.0
-    - dnspython >=1.16.0
+    - protobuf >=3.11.0,<=3.20.1
+    - dnspython >=1.16.0,<=2.1.0
```

Comment:
These dependencies were not correct, see:
https://github.com/mysql/mysql-connector-python/blob/8.0.31/setup.py#L142-L144

I'm seeing `pip check` failing in https://github.com/conda-forge/apache-airflow-providers-mysql-feedstock/pull/14 because of this.